modules/getting-started/proc-customize-rhdh-tech-radar-page.adoc

```diff
@@ -57,17 +77,16 @@ proxy:
       # Change to "false" in case of using self hosted cluster with a self-signed certificate
       secure: true
 ----
-
-Ensure that the API request call returns the response in JSON format.
---
-
-. Define the `TECHRADAR_DATA_URL`` as `pass:c[http://<SERVICE_NAME>/tech-radar]`, for example `pass:c[http://rhdh-customization-provider/tech-radar]`.
+where the `TECHRADAR_DATA_URL` is defined as `pass:c[http://<SERVICE_NAME>/tech-radar]`, for example, `pass:c[http://rhdh-customization-provider/tech-radar]`.
 +
---
 [NOTE]
 ====
```

Comment:
@linfraze Similar to previous common about avoiding admonition close to each other.